src/tests.rs

```diff
@@ -333,6 +333,18 @@ fn test_rpm_header() -> Result<(), Box<dyn std::error::Error>> {
 
     // Verify that if there are no capabilities set then the caps field is None
```

Comment:
We might want to have a negative test for linktos similar to how this test is a negative test for capabilities.

Reply:
Yep. I've improved the assert macro usage where it expected an empty string for non-symlink entires and a non-empty one for a symlink.